server/src/main/java/io/deephaven/server/partitionedtable/PartitionedTableServiceGrpcImpl.java

```diff
@@ -46,18 +46,15 @@ public class PartitionedTableServiceGrpcImpl extends PartitionedTableServiceGrpc
     private final TicketRouter ticketRouter;
     private final SessionService sessionService;
     private final PartitionedTableServiceContextualAuthWiring authWiring;
-    private final TicketResolver.Authorization authorizationTransformation;
```

Comment:
could you clean imports in this file? they aren't wrong by your change, but they are wrong